src/test/scala/com/wavesplatform/TransactionGen.scala

```diff
@@ -178,6 +182,19 @@ trait TransactionGen {
     (assetId, sender, _, amount, timestamp, feeAssetId, feeAmount, attachment) <- transferParamGen
   } yield TransferTransaction.create(assetId, sender, sender, amount, timestamp, feeAssetId, feeAmount, attachment).right.get
 
+  val massTransferGen = (
+      for {
+        (assetId, sender, _, _, timestamp, _, feeAmount, attachment) <- transferParamGen
+        maxTransfers = MassTransferTransaction.MaxTransferCount
```

Comment:
Make this Gen.choose(0, MassTransferTransaction.MaxTransferCount)?
Can it be zero? Looks like it can now. If this should be disabled, consider adding validation or use `NonEmptyList[T]`